src/types/input-configs.ts

```diff
@@ -55,9 +55,15 @@ export interface InputTextareaConfig extends InputBaseConfig, TextareaConfig, Re
     type: "textarea";
 }
 
+export interface CheckboxConfig extends InputBaseConfig {
+    checkedText?: string;
+    uncheckedText?: string;
+}
+
```

Comment:
And once you add it, you should export it from https://github.com/qontu/ngx-inline-editor/blob/master/src/configs.ts